### PR TITLE
Change stories strategy

### PR DESCRIPTION
**Pull request by mh-alahdadian**

### CODE REVIEW

File: packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx [hunk 1]
```diff
@@ -8,10 +22,27 @@ import React, {
...
 } from 'react';
-import { CustomTextFieldProps, ToggleButton, ToggleButtonInput } from '../..';
-import { storyWrapperDecorator } from '../decorators';
-import { LeftInput } from './left-input';
-import { RightInput } from './right-input';
+import { storyWrapperDecorator } from 'src/@storybook/decorators';
+// import { LeftInput } from 'src/@storybook/stories/left-input';
+// import { RightInput } from 'src/@storybook/stories/right-input';
+
+console.log(outlinedInputClasses.notchedOutline);
```
Comment (AmirABody) [inline packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx:29]: `console.log`:)))

File: packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx [hunk 2]
```diff
@@ -8,10 +22,27 @@ import React, {
   useRef,
   useState,
 } from 'react';
-import { CustomTextFieldProps, ToggleButton, ToggleButtonInput } from '../..';
-import { storyWrapperDecorator } from '../decorators';
-import { LeftInput } from './left-input';
-import { RightInput } from './right-input';
+import { storyWrapperDecorator } from 'src/@storybook/decorators';
+// import { LeftInput } from 'src/@storybook/stories/left-input';
+// import { RightInput } from 'src/@storybook/stories/right-input';
```
Comment (AmirABody) [inline packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx:27]: delete these two comments!

File: packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx (hunk 1 above)
Comment (ghost) [inline packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx]: delete `console.log`

File: packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx (hunk 2 above)
Comment (ghost) [inline packages/mui/src/toggle-button-input/toggle-button-input.stories.tsx]: delete these two comments!

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by m-davoodi — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**